multimodal/src/autogluon/multimodal/data/utils.py

```diff
@@ -425,3 +443,97 @@ def get_text_token_max_len(provided_max_len, config, tokenizer, checkpoint_name)
         max_len = min(provided_max_len, default_max_len)
 
     return max_len
+
+
+def construct_processor(
```

Comment:
Maybe construct_image_processor is clearer.

Reply:
thanks for the suggestion, will revise it.